test/api/CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ macro(add_dependencies_to_test target)
       target_link_libraries(${target}
           maliput::api
           maliput::common
+          maliput::geometry_base
```

Comment:
Why does api require geometry_base?

Reply:
It does not, I was trying something and forgot to delete that, thanks.